components/carousel/utils/useCarousel.ts

```diff
@@ -82,18 +82,26 @@ export const useCarouselNftEvents = ({ type }: Types) => {
   const { data: dataRmrk } = useChainEvents('rmrk', type)
   const { data: dataSnek } = useChainEvents('snek', type)
   const { data: dataRmrk2 } = useChainEvents('ksm', type)
+  const { data: dataStmn } = useChainEvents('stmn', type)
 
   const nfts = ref<CarouselNFT[]>([])
 
   // currently only support rmrk and snek
   // moonriver: https://github.com/kodadot/nft-gallery/issues/3891
-  watch([dataRmrk, dataSnek, dataBsx, dataRmrk2], async () => {
+  watch([dataRmrk, dataSnek, dataBsx, dataRmrk2, dataStmn], async () => {
     const rmrkNfts = await flattenNFT(dataRmrk.value, 'rmrk')
     const snekNfts = await flattenNFT(dataSnek.value, 'snek')
     const bsxNfts = await flattenNFT(dataBsx.value, 'bsx')
     const rmrk2Nfts = await flattenNFT(dataRmrk2.value, 'ksm')
-
-    const data = [...rmrkNfts, ...bsxNfts, ...snekNfts, ...rmrk2Nfts]
+    const stmnNfts = await flattenNFT(dataStmn.value, 'stmn')
+
+    const data = [
+      ...rmrkNfts,
```

Comment:
I would change order to
1. stmn
2. rmrk2
3. snek (it shows only for testing right)
4. bsx
5. rmrk